index.html

```diff
@@ -0,0 +1,154 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta
+      name="viewport"
+      header__buttoncontent="width=device-width, initial-scale=1.0"
+    />
+    <link rel="stylesheet" href="./css/style.css" />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20,500,0,-25"
+    />
+    <title>Medical core</title>
+  </head>
+  <body>
+    <!-- start navbar -->
+    <nav class="header">
+      <div class="menu">
+        <div class="header__icon">
+          <h1 class="header__name">Medical core</h1>
+        </div>
+        <ul class="header__links">
+          <li class="header_link"><a href="#">Home</a></li>
+          <li class="header_link"><a href="#">Product</a></li>
+          <li class="header_link"><a href="#">Precing</a></li>
+          <li class="header_link"><a href="#">Contact</a></li>
+        </ul>
+      </div>
+      <div class="btn__container">
+        <button class="btn">Login</button>
+        <button class="btn btn--active">
+          JOIN US
+          <span class="btn__icon"
+            ><img src="./media/Vector.png" alt="vector-icon"
+          /></span>
+        </button>
+      </div>
+    </nav>
+    <!--End navbar  -->
+
+    <!-- Feeds start -->
+    <section>
+      <div class="primary-background">
+        <div class="feeds__contents feeds__contents--start">
+          <p class="feeds__title primary-color">Welcome</p>
+          <h1 class="feeds__subtitle">
+            A Great Place to <br />
+            Receive Care
+          </h1>
+          <p class="paragraph">Get your best looking smile now</p>
+          <div class="action__button">
+            <button class="btn btn--active">Join us</button>
+            <button class="btn btn--no-active">Learn More</button>
+          </div>
+        </div>
+      </div>
+
+      <div class="feeds secondary-background">
+        <img
+          src="./media/medical-core-removebg-preview.png"
+          alt=""
+          class="feeds__image"
+        />
+        <div class="feeds_right">
+          <h2 class="feeds__subtitle">
+            <span class="feeds__linear">Meet</span> Our Experts
+          </h2>
+          <p class="paragraph">
+            Problems trying to resolve the conflict between the two major realms
+            of Classical physics: Newtonian mechanics
+          </p>
+          <div>
+            <button class="btn btn--feeds">
```

Comment:
btn--feeds is it a modifier should be named  in a way that describe its usage(What it is doing)